http/src/main/java/io/micronaut/http/body/ContextlessMessageBodyHandlerRegistry.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2017-2023 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.http.body;
+
+import io.micronaut.core.annotation.Experimental;
+import io.micronaut.core.annotation.Internal;
+import io.micronaut.core.io.buffer.ByteBufferFactory;
+import io.micronaut.core.type.Argument;
+import io.micronaut.http.MediaType;
+import io.micronaut.runtime.ApplicationConfiguration;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+/**
+ * {@link MessageBodyHandlerRegistry} implementation that does not need an application context.
+ *
+ * @author Jonas Konrad
+ * @since 4.0.0
+ */
+@Internal
+@Experimental
+public final class ContextlessMessageBodyHandlerRegistry extends RawMessageBodyHandlerRegistry {
```

Comment:
Since this is only used by the client can it be moved to the client module and made package private?

Reply:
RawMessageBodyHandlerRegistry is package-private so moving ContextlessMessageBodyHandlerRegistry would need to make RawMessageBodyHandlerRegistry public